src/components/common/UserProfile/UserProfile.tsx

```diff
@@ -15,7 +15,7 @@ const displayName = 'common.UserProfile';
 
 const UserProfile = () => {
   const { username } = useParams();
-  const { user, loading, error } = useUserByName(username || '');
+  const { user, loading, error } = useUserByNameOrAddress(username || '');
```

Comment:
I would also change the name of this param to something more general as with this PR now this could not be just a username.